modules/enabling-scanner-v4-helm.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-helm_{context}"]
+= Enabling RHACS Scanner V4 when installing with Helm
+
+[role="_abstract"]
+Scanner V4 is not enabled by default, but you can enable it during installation.
+
+If you are installing {product-title-short} on secured clusters by using Helm charts, you must first enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. Run the command to install {product-title-short} with customizations, following the instructions in "Install Central using Helm charts with customizations":
++
+[source,terminal]
+----
+helm install -n stackrox --create-namespace \
+  stackrox-central-services rhacs/central-services \
+  --set scannerV4.disable=false \
+  -f <path_to_values_public.yaml> -f <path_to_values_private.yaml>
+----
+. If Scanner V4 is successfully enabled, the configuration summary displayed is similar to the following example:
```

Comment:
This seems to describe the result of an action, and does not provide an instruction for users, which [each step needs to do](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps). I would just remove the list item here and nest the line within the previous step:
 ```suggestion
+
If Scanner V4 is successfully enabled, the configuration summary displayed is similar to the following example:
```

Reply:
Or if this output is expected to appear automatically after running the above command, you could also omit the text and just use the example output title:
```suggestion
+
.Example output
```